edx-platform/pearson-aws-theme/lms/static/sass/partials/lms/theme/_login-register.scss

```diff
@@ -1,12 +1,71 @@
+$form-background: #3e5cd2;
+$bg-image: '#{$static-path}/images/login-register.jpg';
+
+
 .login-register-content {
+  min-height: 100vh;
+  background-image: url($bg-image);
+  background-position: left bottom;
```

Comment:
Is it necessary to specify this rule? @kuipumu

Reply:
I used min-height to increase the content height to show more of the background image, not sure what could be a more appropriate height, and background-position is just to position where I thought looked better

Reply:
But is it necessary to specify "background-position"? @kuipumu

Reply:
@Squirrel18 No, it's only an aesthetic choice

Reply:
Then remove it, to avoid unnecessary rules. @kuipumu